boards/arm/holyiot_yj16019/doc/index.rst

```diff
@@ -34,8 +34,8 @@ contains the processor's information and the datasheet.
 Hardware
 ********
 
-The nRF52832 of the Holyiot YJ-16019 is clocked by an external crystal
-with a frequency of 32.768 kHz.
+The nRF52832 of the Holyiot YJ-16019 is clocked by an external crystal with a frequency of 32 MHz
+(Y1). The 32.768 kHz crystal (Y2) shown on the board schematics is not mounted.
```

Comment:
```suggestion
(X1). The 32.768 kHz crystal (Y2) shown on the board schematics is not mounted.
```

Reply:
But the part is designated Y1 on the schematics?

Reply:
I looked at the schematic downloaded from [here](http://www.holyiot.com/ebxw_list.asp?pages=4&xwlb_id=) and it said X1:
![yj-16019](https://user-images.githubusercontent.com/16913290/173504227-be334ccb-78b0-4b16-aaee-411b8b5476f8.png)
so I thought that this Y1 was a typo.